amphora-image-builder/entrypoint.sh

```diff
@@ -11,6 +11,18 @@ git clone --depth=1 https://github.com/openstack/octavia
 DIB_REPO_PATH="$(pwd)/octavia"
 export DIB_REPO_PATH
 
+# run local config elements
+git clone https://github.com/apdibbo/cloud-image-builders.git
+#git clone https://github.com/stfc/cloud-image-builders.git
```

Comment:
Just checking, is this meant to be pointing at a fork?

Reply:
Not anymore :)